packages/core/CHANGELOG.md

```diff
@@ -3,6 +3,15 @@
 All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
+## [1.3.0](https://github.com/coinbase/rest-hooks/compare/@rest-hooks/core@1.2.1...@rest-hooks/core@1.3.0) (2021-05-30)
```

Comment:
do version bumps usually happen in PRs, or in tagged publish commits?

Reply:
https://github.com/coinbase/rest-hooks/pull/875 hasn't been approved so i'm working off my local master that doesn't match remote